pygmo/core.cpp

```diff
@@ -959,8 +923,25 @@ BOOST_PYTHON_MODULE(core)
 
     // Exposition of stand alone functions
     // Multi-objective utilities
-    bp::def("fast_non_dominated_sorting", fast_non_dominated_sorting_wrapper,
-            pygmo::fast_non_dominated_sorting_docstring().c_str(), bp::arg("points"));
+    bp::def("fast_non_dominated_sorting",
+            +[](const bp::object &x) -> bp::object {
```

Comment:
was this not a good example of where a wrapper may be better? Long lambdas are weird no?